projects/observability/src/pages/apis/service-detail/service-detail.component.test.ts

```diff
@@ -7,16 +7,28 @@ import { FeatureState, FeatureStateResolver } from '@hypertrace/common';
 import { GRAPHQL_OPTIONS } from '@hypertrace/graphql-client';
 import { ENTITY_METADATA } from '@hypertrace/observability';
 
+import { NavigableTab } from '@hypertrace/components';
 // tslint:disable-next-line: import-blacklist
 import { entityMetadata } from '../../../../../../src/app/entity-metadata';
 import { ServiceDetailComponent } from './service-detail.component';
 import { ServiceDetailModule } from './service-detail.module';
 
+const featureFlaggedTabs: NavigableTab[] = [
+  {
+    path: 'instrumentation',
+    label: 'Instrumentation Quality'
+  },
+  {
+    path: 'deployments',
+    label: 'Deployments'
+  }
+];
+
 describe('ServiceDetailComponent', () => {
   let component: ServiceDetailComponent;
   let fixture: ComponentFixture<ServiceDetailComponent>;
 
-  beforeEach(() => {
+  test("Component exists and doesn't display Instrumentation tab when feature flag is disabled", () => {
```

Comment:
Looks like this test has been changed to check if feature flagged tabs are displayed when enabled. Please update the description accordingly.

Reply:
done